_layouts/doc.html

```diff
@@ -3,7 +3,7 @@
 <main class="doc">
   <article class="container">
     {% include ad.html %}
-    {{ content }}
+    {{ content | replace: '(recommended)', '<span aria-label="recommended" class="glyphicon glyphicon-ok"></span>' | replace: '(fixable)', '<span aria-label="fixable" class="glyphicon glyphicon-wrench"></span>' }}
```

Comment:
Locally, yes! Here are [the docs](https://github.com/Shopify/liquid/wiki/Liquid-for-Designers#standard-filters) for Liquid filters if you're interested. Without the changes to index.md it replaces the instances of the words in the intro too, so if we want to implement the original design with the legend I'll have to make another small PR to the main repo.